clients/snowflake/tableid.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/artie-labs/transfer/lib/sql"
 )
 
-var dialect = sql.SnowflakeDialect{UppercaseEscNames: true}
+var dialect = sql.SnowflakeDialect{}
```

Comment:
This isn't using legacy mode because we don't need to escape any more tables right?

Reply:
Yep, we've been escaping all Snowflake tables for some time.